### ISSUE TITLE
.addRule with @font-face does not work the second time

### ISSUE THREAD

**Issue by geigerzaehler** [cssinjs/jss#1262]
__Expected behavior:__
Using the following code I’d expect the stylesheet to contain _two_ `@font-face` declrations, one for each call of `.addRule`.
~~~js
const sheet = jss.createStyleSheet()
sheet.addRule('@font-face', { name: 'font1' })
sheet.addRule('@font-face', { name: 'font2' })
sheet.attach()
~~~

__Describe the bug:__
The stylesheet contains only the first `@font-face` declration.

__Codesandbox link:__ 
https://codesandbox.io/s/xenodochial-chatelet-rpqrc

__Versions (please complete the following information):__
 - jss: 10.0.3 (was working in 10.0.2)
 - Browser [e.g. chrome, safari]: all
 - OS [e.g. Windows, macOS]: all

**Comment by kof** [comment]
will be fixed in 10.0.5 or higher

**Comment by kof** [comment]
released in 10.1.0
